media/css/pebbles/includes/mixins/_typography.scss

```diff
@@ -0,0 +1,81 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+// Font families
+@mixin fira-sans {
+    font-family: $font-fira-sans;
+}
+
+@mixin fira-sans-light {
+    font-family: 'Fira Sans Light', X-LocaleSpecific-Light, $font-fira-sans;
```

Comment:
Fira does have a really broad character set, but some languages that Fira still can't render (because it's a totally different writing system) will probably still want to specify their own font. Arabic, Japanese, etc. I think we should keep it.